### PR TITLE
Revert "Revert "PMM-5777 Inventory add external exporter endpoint.""

### PR DESCRIPTION
**Pull request by AlekSi**
Reverts percona/pmm#522

### CODE REVIEW

**Comment by BupycHuk** [comment]
What's going on?

**Comment by AlekSi** [comment]
https://github.com/percona/pmm-managed/pull/377 and other PRs should use that branch, and be merged together

File: api/inventorypb/agents.proto [hunk 1]
```diff
@@ -325,6 +325,28 @@ message RDSExporter {
...
+  string agent_id = 1;
+  // Node identifier where this instance runs.
+  string runs_on_node_id = 2;
+  // If disabled, metrics from this exporter will not be collected.
+  bool disabled = 3;
+  // Service identifier.
+  string service_id = 4;
+  // HTTP basic auth username for collecting metrics.
+  string username = 5;
+  // Scheme to generate URI to exporter metrics endpoints(default: http).
```
Comment (AlekSi) [inline api/inventorypb/agents.proto:340]: But that structure is returned, and that field is always set, isn't it? So there is no default?

File: api/inventorypb/agents.proto [hunk 2]
```diff
@@ -325,6 +325,28 @@ message RDSExporter {
...
+  string scheme = 6;
+  // Path under which metrics are exposed, used to generate URI(default: /metrics).
```
Comment (AlekSi) [inline api/inventorypb/agents.proto:342]: The same as above.

File: api/inventorypb/agents.proto [hunk 3]
```diff
@@ -325,6 +325,28 @@ message RDSExporter {
...
+  string metric_path = 7;
```
Comment (AlekSi) [inline api/inventorypb/agents.proto:343]: Let's call it `metrics_path` while can

File: api/inventorypb/agents.proto [hunk 4]
```diff
@@ -893,6 +917,53 @@ message ChangeRDSExporterResponse {
...
+  // Service identifier.
+  string service_id = 2;
+  // HTTP basic auth username for collecting metrics.
+  string username = 3;
+  // HTTP basic auth password for collecting metrics.
+  string password = 4;
+  // Scheme to generate URI to exporter metrics endpoints.
+  string scheme = 6;
+  // Path under which metrics are exposed, used to generate URI.
+  string metric_path = 7;
```
Comment (AlekSi) [inline api/inventorypb/agents.proto:938]: `metrics_path`

**Pull request merged**